reactor-netty-http/src/test/java/reactor/netty/http/Http2Tests.java

```diff
@@ -149,6 +155,40 @@ void testIdleTimeoutWithHttp11AndH2cHandlesRequestsCorrectly() {
 				.verifyComplete();
 	}
 
+	@Test
+	void testIdleTimeoutWithHttp11AndH2cHandlesHttp11RequestsCorrectly() {
```

Comment:
I see that you copied `testIdleTimeoutWithHttp11AndH2cHandlesRequestsCorrectly`
Wdyt if we move this test to `reactor.netty.http.HttpProtocolsTests` and run it with `@ParameterizedCompatibleCombinationsTest` thus we will test that the fix is working for all compatible combinations for `HttpServer` and `HttpClient`?

Reply:
Oh I see, really better place to have this test.